components/viz/host/host_gpu_memory_buffer_manager.cc

```diff
@@ -475,9 +475,15 @@ void HostGpuMemoryBufferManager::OnGpuMemoryBufferAllocated(
 bool HostGpuMemoryBufferManager::CreateBufferUsesGpuService(
     gfx::BufferFormat format,
     gfx::BufferUsage usage) {
-  return gpu_memory_buffer_support_->GetNativeGpuMemoryBufferType() !=
+// hack for gpu memory allocation
+  return false;
+#if 0
+  bool res = gpu_memory_buffer_support_->GetNativeGpuMemoryBufferType() !=
              gfx::EMPTY_BUFFER &&
          IsNativeGpuMemoryBufferConfiguration(format, usage);
+
+  return res;
+#endif
```

Comment:
This is probably needed/happening because we are taking  
the IS_LINUX path in [1] - we should add an IS_STARBOARD
entry there (//gpu is a low level folder, like //base  
or //media, so can know about "starboard").



[1] https://source.chromium.org/chromium/chromium/src/+/main:gpu/ipc/common/gpu_memory_buffer_support.cc;l=52-64?q=GetNativeGpuMemoryBufferType&ss=chromium